### PR TITLE
Corrected spacing and updated plot options to new system in Points (Scatter) Plot dialog

### LINKED ISSUE #7556: Change the specific plots to the new system
I am suggesting @EstherNjeriLiberatta for this, as (I am afraid) you did the PICSA and mapping graphs very well!

a) I suggest there be a single button, just `Plot Options`.  And it has the same pull-down as for the mapping dialogue.  
b) Then the first option is also Plot Options.
c) This is followed by each of the remaining options for that dialogue, and they could be in alphabetical order.
d) I suggest they don't change for the different options at the top.  So, for example, for bar charts it is the same whether the top radio button if Frequency, Value, Treemap or Wordcloud.
e) Then simply include each of those, including label and lollipop, etc.

### LINKED ISSUE #7573: Correct spacing between the Point(Scatter) plot dialogue.
**Describe the bug**
This is not a bug but a request for change in the dialogue design. There appears to be too much space between the dialogue.

**To Reproduce**
1. Go to 'Describe menu'
2. Click on 'Specific Tables/Graphs'
3. Scroll down to 'Point(Scatter) Plot' 
4. See dialogue

**Screenshots**
image

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
Fixes #7573 
Fixes (partially) #7556
In this PR, I have replaced the` Point Options` button in the Scatter plot dialogue with the new `Plot Options` button which includes a drop-down and made minor design improvements.
@rdstern @africanmathsinitiative/developers This is ready for review.

### CODE REVIEW

**Comment by N-thony** [comment]
> Fixes #7573 Partially fixes #7556 In this PR, I have replaced the` Point Options` button in the Scatter plot dialogue with the current `Plot Options` button which includes a drop-down and made minor design improvements. @rdstern @africanmathsinitiative/developers This is ready for review.

@Amsale2022 please could you test?

**Review by rdstern — CHANGES_REQUESTED** [review]
a) The spacing has given a problem here.  It now says `Add Line of Best`  and the word `fit ` is no longer showing.  Once corrected, check also there is room for the whole text, once translated into French.  Perhaps the dialogue can't be made so narrow.
b) There is an option of adding a rug plot.  So the plot pull down should add rug plot as well.

**Comment by N-thony** [comment]
> a) The spacing has given a problem here. It now says `Add Line of Best` and the word `fit ` is no longer showing. Once corrected, check also there is room for the whole text, once translated into French. Perhaps the dialogue can't be made so narrow? b) Ther line of best fit is a geom smooth. So that should be added to the pull-down. The pull-down should include all the options directly from the main dialogue. c) There is an option of adding a rug plot. So the plot pull down should add rug plot as well.

@EstherNjeriLiberatta can you make these changes? Thanks.

**Comment by EstherNjeriLiberatta** [comment]
@rdstern Please review these changes. I've widened the dialogue to allow the french translation to be seen. The rug and smooth options are now added in the drop-down. Enabled only when their respective check boxes are checked. Thank you.

**Comment by N-thony** [comment]
> @rdstern Please review these changes. I've widened the dialogue to allow the french translation to be seen. The rug and smooth options are now added in the drop-down. Enabled only when their respective check boxes are checked. Thank you.

@rdstern  please have a look.

**Review by N-thony — DISMISSED** [review]

File: instat/dlgScatterPlot.vb [hunk 1]
```diff
@@ -281,23 +304,65 @@ Public Class dlgScatterPlot
...
+        Next
     End Sub
 
-    Private Sub ucrChkLineofBestFit_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrChkLineofBestFit.ControlValueChanged
-        If ucrChkLineofBestFit.Checked Then
-            clsBaseOperator.AddParameter("geom_smooth", clsRFunctionParameter:=clsGeomSmoothFunc, iPosition:=4)
-        Else
-            clsBaseOperator.RemoveParameterByName("geom_smooth")
-        End If
+    Private Sub toolStripMenuItemSmoothOptions_Click(sender As Object, e As EventArgs) Handles toolStripMenuItemSmoothOptions.Click
```
Comment (lloyddewit) [inline instat/dlgScatterPlot.vb:336]: This sub seems very similar to the sub above.
Move code to a private function?

**Review by lloyddewit — APPROVED** [review]

**Review by lloyddewit — CHANGES_REQUESTED** [review]
I think I approved by mistake, please see my comments above

File: instat/dlgScatterPlot.vb [hunk 2]
```diff
@@ -281,23 +304,65 @@ Public Class dlgScatterPlot
...
+        'The aesthetics parameters on the main dialog are repopulated as required. 
+        For Each clsParam In clsRaesFunction.clsParameters
+            If clsParam.strArgumentName = "x" Then
+                If clsParam.strArgumentValue = Chr(34) & Chr(34) Then
+                    ucrReceiverX.Clear()
+                Else
+                    ucrReceiverX.Add(clsParam.strArgumentValue)
+                End If
+                'In the y case, the vlue stored in the clsReasFunction in the multiplevariables case is "value", however that one shouldn't be written in the multiple variables receiver (otherwise it would stack all variables and the stack ("value") itself!).
+                'Warning: what if someone used the name value for one of it's variables independently from the multiple variables method ? Here if the receiver is actually in single mode, the variable "value" will still be given back, which throws the problem back to the creation of "value" in the multiple receiver case.
```
[the remark this answers is not in the document]
Comment (EstherNjeriLiberatta) [reply instat/dlgScatterPlot.vb:323]: @lloyddewit it is not my comment and I'm not sure about this can you help with rephrasing this?

**Comment by EstherNjeriLiberatta** [comment]
@lloyddewit Thank you for reviewing. I have made the requested changes.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@EstherNjeriLiberatta I pushed a change to try and improve the readability of the `EnableDisableOptions()` function. This also resolves the open comment above.
Please could you review my change?

@rdstern If you can test/approve, then we can merge

Thanks!

**Review by rdstern — APPROVED** [review]
Excellent

**Comment by lloyddewit** [comment]
> @EstherNjeriLiberatta I pushed a change to try and improve the readability of the `EnableDisableOptions()` function. This also resolves the open comment above. Please could you review my change?

@EstherNjeriLiberatta I went ahead and merged but if you could find time to review the change, then I'd appreciate. If needed, I can implement any suggestions in a new PR.

**Pull request merged**